main.js

```diff
@@ -23,31 +26,55 @@ var activitySeconds = document.querySelector('.activity-seconds');
 var startActivityButton = document.querySelector('.start-activity');
 var startStopButton = document.querySelector('.start-stop-button');
 
-// ======================================
+var timerActivityDescription = document.querySelector('.timer-activity-description');
+var timer = document.querySelector('.time');
 
+startActivityButton.addEventListener('click', startActivityFunc)
 
 // event listener for buttons (change color of font and change image)
 allCategoryButtons.addEventListener('click', function (event) {
   if (event.target.className === 'icon-study' || event.target.className === 'icon study') {
-    document.querySelector('img.study').classList.toggle('hidden')
-    document.querySelector('img.study-active').classList.toggle('hidden')
-    currentActivity = 'Study'
+    selectButton('study');
+    deselectButton('exercise');
+    deselectButton('meditate');
   } else if (event.target.className === 'icon-exercise' || event.target.className === 'icon exercise') {
-    document.querySelector('img.exercise').classList.toggle('hidden')
-    document.querySelector('img.exercise-active').classList.toggle('hidden')
-    currentActivity = 'Excercise';
+    deselectButton('study');
+    selectButton('exercise');
+    deselectButton('meditate');
   } else if (event.target.className === 'icon-meditate' || event.target.className === 'icon meditate') {
-    document.querySelector('img.meditate').classList.toggle('hidden')
-    document.querySelector('img.meditate-active').classList.toggle('hidden')
-    currentActivity = 'Meditate'
+    deselectButton('study');
+    deselectButton('exercise');
+    selectButton('meditate');
   }
```

Comment:
Such a great use of helper functions! Thank you for making this so much more functional!